cfme/tests/cli/test_appliance_cli.py

```diff
@@ -30,13 +30,12 @@
 
 
 @requires_59
-def test_appliance_console_cli_datetime(temp_appliance_preconfig_funcscope):
+def test_appliance_console_cli_datetime(appliance):
     """Grab fresh appliance and set time and date through appliance_console_cli and check result"""
-    app = temp_appliance_preconfig_funcscope
-    app.ssh_client.run_command("appliance_console_cli --datetime 2020-10-20T09:59:00")
+    appliance.ssh_client.run_command("appliance_console_cli --datetime 2020-10-20T09:59:00")
```

Comment:
This test isn't resetting the datetime configuration, which is why it was using temp_appliance_preconfig_funcscope.  If that fixture is failing to pull a sprout appliance, we need to resolve that. I think using appliance here will have negative impacts on tests that follow.

Reply:
Not sure where hostname assertion comes into play here in this test function?  I see in the test below hostname is used changed and asserted against - but again isn't reset after the test is done, because the test was running on a one-off appliance.  We need to resolve the core failures here when using one-off appliances, or need to reliably reset the changes made in these tests.